documentation/man/features/init.rst

```diff
@@ -10,6 +10,40 @@ SYNOPSIS
 
 DESCRIPTION
 ===========
-This command creates a **kworkflow.config** file in the current directory. The
-primary reason for running kw init is to pick up a freshly created config
-file.
+This command creates a **.kw** folder containing a kworkflow.config file in the
```

Comment:
Enclose `kworkflow.config` in double asterisks, do this for files and folder names.